raiden/transfer/channel.py

```diff
@@ -2302,10 +2300,31 @@ def handle_channel_settled(
     return TransitionResult(channel_state, events)
 
 
+def update_fee_schedule_after_balance_change(
+    channel_state: NettingChannelState, fee_config: MediationFeeConfig
+) -> List[Event]:
+    if not channel_state:
```

Comment:
Can this ever be `None`? According to the type not.

Reply:
You're right. This was left over from the previous implementation. I'll remove it.

Reply:
This makes me want to investigate mypy's `--warn-unreachable` flag.

Reply:
Lefteris said it had false positives when he tried.